test/WebSites/AddServicesWebSite/Startup.cs

```diff
@@ -9,6 +9,8 @@ public class Startup
     {
         public void Configure(IApplicationBuilder app)
         {
+            app.UseIISPlatformHandler();
+            
```

Comment:
@Kagamine sorry line 13 has 12 spaces on it (indentation) - they just need to be trimmed. Keep the empty line, though.